src/tasks/intern.ts

```diff
@@ -1,26 +1,46 @@
 import global from '@dojo/shim/global';
+import { assign } from '@dojo/core/lang';
 
 import Node from '../lib/executors/Node';
 import { Config } from '../lib/executors/Executor';
+import { getConfig } from '../lib/node/util';
 
 interface TaskOptions extends grunt.task.ITaskOptions, Partial<Config> {
 	[key: string]: any;
 }
 
+function configure(options: TaskOptions): Promise<{
+	config: Partial<Config>;
+	options: TaskOptions;
+}> {
+	if (options.config) {
+		return getConfig(options.config, []).then(({ config }) => {
+			delete options.config;
```

Comment:
Instead of deleting config from the passed-in options, it'd be nicer to return a new object that didn't have the config property (since we're returning options anyway).